_dev/src/components/product-feed/product-feed-card.vue

```diff
@@ -295,10 +300,12 @@ export default defineComponent({
   },
   methods: {
     startConfiguration() {
+      const step = Object.values(ProductFeedSettingsPages)[this.getActiveStep - 1];
```

Comment:
Can you please get a confirmation from the Product about this behavior?

Reply:
I need this because the stepper starts at 1, but when I want to search the reference in the enum with an index, I have to make a -1 because the beginning of the enum starts at 0

Reply:
I mean the change where you do not start over, but from the active step